### DISCUSSION TITLE
Intermittent "Can't reach database server at" error

### DISCUSSION THREAD

**Discussion by kbariotis** [prisma/prisma#18226] [Q&A] [answered]
We've been getting lots of "Can't reach database server at" error from time to time in different modules. During that time, the DB (Postgres/RDS) is reporting lots of "Could not receive data from client: Connection reset by peer".

The RDS monitor shows us that we haven't reached the connection limit and using the [Prisma metrics](https://www.prisma.io/docs/concepts/components/prisma-client/metrics) module, we haven't been reaching the maximum connection timeout.

What could be causing this issue? Any help to debug it? Thank you

**Error:**
```
Invalid `prisma.***.findFirst()` invocation:


Can't reach database server at ***

Please make sure your database server is running at ***.
```

**Comment by ludralph** [comment, marked as answer]
Hi @kbariotis 👋 

This error could occur because of many reasons. I advise reading this troubleshooting guide for [Postgres/RDS](https://aws.amazon.com/premiumsupport/knowledge-center/rds-aurora-postgresql-connection-errors/). This error appeared because your client application dropped the connection to the database and postgres logged it. So either fix the client or look at your network to see if it is flakey.

**Reply by ludralph** [reply]
Hi @kbariotis 👋 

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
